src/others/contexts/api.tsx

```diff
@@ -12,50 +14,25 @@ export const queryClient = new QueryClient({
   },
 });
 
-export const API_DOMAIN = process.env.REACT_APP_API_DOMAIN || "http://127.0.0.1";
-
-export type ID = string;
-
-export type ServiceType = "external" | "supplies" | "chat";
-
-export interface Service {
-  id: ID;
-  name: string;
-  type: ServiceType;
-  path: string;
-}
-
-export interface CountryInfo {
-  services: {
-    inhouse: Service[];
-    external: Service[];
-  };
-}
-
-export interface Country {
-  id: ID;
-  name: string;
-  info: CountryInfo;
-}
+export const API_DOMAIN = process.env.REACT_APP_API_DOMAIN || 'http://127.0.0.1';
 
 export function useCountriesQuery() {
   const { i18n } = useTranslation();
 
   return useQuery<Country[]>(`locationQuery${i18n.language}`, async () => {
     try {
-      const result = await fetch(`${API_DOMAIN}/countries?locale=${i18n.language}`)
+      const result = await fetch(`./assets/dataset.csv`)
         .then((res) => {
           if (!res.ok) throw new Error(res.statusText);
 
           return res;
         })
-        .then((res) => res.json());
+        .then((res) => res.text());
```

Comment:
why do you mix async/await with Promise apis?

Reply:
Just to keep this function as close to the original as possible, but I agree, updated!